rerun_js/web-viewer-react/index.js

```diff
@@ -28,7 +28,7 @@ import * as rerun from "@rerun-io/web-viewer";
  */
 export default class WebViewer extends React.Component {
   /** @type {React.RefObject<HTMLDivElement>} */
-  #parent = createRef();
+  #parent = /** @type {React.RefObject<HTMLDivElement>} */ (createRef());
```

Comment:
this annotation was the only thing that broke when switching to React 19. The fix is what the ai suggested and it seems to work just fine 🤷